rates:
  robot_state: 20.0
  metrics: 0.2
  robot_command: 1.0
  power: 1.0
  lease: 1.0
  front_image: 10.0
  side_image: 5.0
  rear_image: 5.0
  estop: 1.0
